charts/newrelic-infra-operator/templates/admission-webhooks/mutatingWebhookConfiguration.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: {{ template "newrelic-infra-operator.fullname" . }}
+{{- if .Values.certManager.enabled }}
+  annotations:
+    certmanager.k8s.io/inject-ca-from: {{ printf "%s/%s-root-cert" .Release.Namespace (include "newrelic-infra-operator.fullname" .) | quote }}
+    cert-manager.io/inject-ca-from: {{ printf "%s/%s-root-cert" .Release.Namespace (include "newrelic-infra-operator.fullname" .) | quote }}
+{{- end }}
+  labels:
+    {{- include "newrelic-infra-operator.labels" . | nindent 4 }}
+webhooks:
+- name: newrelic-infra-operator.newrelic.com
+  clientConfig:
+    service:
+      name: {{ template "newrelic-infra-operator.fullname" . }}
+      namespace: {{ .Release.Namespace }}
+      path: "/mutate-v1-pod"
+{{- if not .Values.certManager.enabled }}
+    caBundle: ""
```

Comment:
@invidian  will we need at the end "reinvocationPolicy: IfNeeded"?

Reply:
I just saw you mentioned "we can fix the remaining stuff later." I guess you were referring to this as well

Reply:
Yes, we should add `reinvocationPolicy: IfNeeded` I think.

Reply:
Added.